src/api/MainSolver.cc

```diff
@@ -95,34 +82,29 @@ bool MainSolver::pop()
         frames.pop();
         substitutions.pop();
         firstNotSimplifiedFrame = std::min(firstNotSimplifiedFrame, frames.frameCount());
-        if (not isLastFrameUnsat()) {
-            getSMTSolver().restoreOK();
-        }
+        if (not isLastFrameUnsat()) { getSMTSolver().restoreOK(); }
         return true;
     }
     return false;
 }
 
-PartitionManager & MainSolver::getPartitionManager() { return pmanager; }
```

Comment:
This is what clang-format did by itself? :o

Reply:
There was a bit of manual intervention here :)
Because it is out-of-line, clang-format did not want to have on one line. Then I realized this is an artifact from my previous changes and that it can safely go back to the header.